src/d3a/setup/strategy_tests/external_ess_bids.py

```diff
@@ -31,7 +31,7 @@ def get_setup(config):
                 'House 1',
                 [
                     Area('PV', strategy=PVStrategy(
-                        max_panel_power_W=2000, initial_selling_rate=30, final_selling_rate=30.0)
+                        max_panel_power_W=2000, initial_selling_rate=40, final_selling_rate=30.0)
```

Comment:
Why is this needed?

Reply:
This was discovered while testing `integration_tests/test_ess_bid_connection.py` on `d3a-api-client` side when the bids placed by ess was instantly matched by `pv` leading to [offered_buy_kWh](https://github.com/gridsingularity/d3a-api-client/pull/113/commits/26893ff1560aa95a4dcbfc57091f7f74ad93ac57#diff-9487872865372fcdd22510280a797481a6ccc35e7cc7f9071cd63006516c2631R32) not equivaled to cumulative placed bid energy as parts of it was already matched by pv

Reply:
Ah, yes, I just ran into this in my tests too. Weird that this only happened now.

Reply:
I'm not sure whether this might be related, but also keep in mind that the `replace_existing` parameter is `True` by default. So each offer/bid will always replace the previous ones unless you specify `replace_existing=False`.

Reply:
@fievelk yes, i just realise that you havent pushed your `replace_existing` -PR here yet. Why is that?

Reply:
@faizan2590 could you please add a TODO comment in here that logs the change, why we did it and that this should be reverted once the `replace_existing`-PR is pushed? Thanks!